src/Microsoft.ML.TimeSeries/ExponentialAverageTransform.cs

```diff
@@ -22,13 +22,14 @@ namespace Microsoft.ML.TimeSeriesProcessing
     /// <summary>
     /// ExponentialAverageTransform is a weighted average of the values: ExpAvg(y_t) = a * y_t + (1-a) * ExpAvg(y_(t-1)).
     /// </summary>
-    public sealed class ExponentialAverageTransform : SequentialTransformBase<Single, Single, ExponentialAverageTransform.State>
+    internal sealed class ExponentialAverageTransform : SequentialTransformBase<Single, Single, ExponentialAverageTransform.State>
     {
         public const string Summary = "Applies a Exponential average on a time series.";
         public const string LoaderSignature = "ExpAverageTransform";
         public const string UserName = "Exponential Average Transform";
         public const string ShortName = "ExpAvg";
 
+#pragma warning disable 0649
         public sealed class Arguments : TransformInputBase
```

Comment:
why do we need this?

Reply:
It will warn for variable not initialized error. This pattern is used across the codebase

Reply:
Why did we not need it before?

---
In reply to: [254478686](https://github.com/dotnet/machinelearning/pull/2344#discussion_r254478686) [](ancestors = 254478686)

Reply:
I think it has to do with making the class internal.


---
In reply to: [254483842](https://github.com/dotnet/machinelearning/pull/2344#discussion_r254483842) [](ancestors = 254483842,254478686)